Tutorial_1c_Line_Ratios.ipynb

```diff
@@ -0,0 +1,736 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    unc_ii_12co = mad_std_map_12co.value*cube_12co_conv_reproj.header['CDELT3']*np.sqrt(n_chan_cube)
</pre><p>If the spectral unit has been updated, you could grab the channel width directly from spectral-cube:</p>

<pre class="ql-syntax" spellcheck="false">chan_width = np.abs(np.diff(cube_12co_conv_reproj[:2])[0])
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>